### PR TITLE
Add simple CLI

### PR DESCRIPTION
**Pull request by manzt**
Adds simple CLI to run benchmark.

```bash
$ npm run bench
```

### CODE REVIEW

**Comment by manzt** [comment]
We could add more robust flag parsing puppeteer specific flags (e.g. `--no_headless`), but for now it just passes any chrome flags.

```bash
./benchmark.js out.csv --disable-http2 --auto-open-devtools-for-tabs # this should work
```

**Comment by manzt** [comment]
@ilan-gold see new args

File: benchmark.js [hunk 1]
```diff
@@ -36,6 +39,17 @@ async function run(outfile, config = {}) {
   })
 }
 
-run('./chrome_http2.csv');
-run('./chrome_http1.csv', { args: ["--disable-http2"] });
+function main() {
+  const args = process.argv.slice(2);
+  if (args.length < 1) {
+    console.log("./benchmark.js <filename> [--no-headless] [--ignore-http-errors] [chrome_flags]")
```
Comment (ilan-gold) [inline benchmark.js]:
```suggestion
    console.log("./benchmark.js <filename> [--no-headless] [--ignore-https-errors] [chrome_flags]")
```

File: benchmark.js [hunk 2]
```diff
@@ -36,6 +39,17 @@ async function run(outfile, config = {}) {
...
+    process.exit(1);
+  }
+  const outfile = args[0];
+  const flags = new Set(args.slice(1));
+  const headless = !flags.delete('--no-headless');
+  const ignoreHTTPErrors = flags.delete('--ignore-http-errors');
+  run(outfile, { headless, ignoreHTTPErrors, args: [...flags] });
```
Comment (ilan-gold) [inline benchmark.js]:
```suggestion
  const ignoreHTTPSErrors = flags.delete('--ignore-https-errors');
  run(outfile, { headless, ignoreHTTPSErrors, args: [...flags] });
```

File: package.json [hunk 3]
```diff
@@ -5,7 +5,10 @@
   "main": "build/index.js",
   "scripts": {
     "start": "webpack-dev-server --hot",
-    "build": "webpack -p"
+    "build": "webpack -p",
+    "bench:h1": "./benchmark.js chrome_http1.csv --disable-http2 --ignore-http-errors --no-headless",
+    "bench:h2": "./benchmark.js chrome_http2.csv --ignore-http-errors --no-headless",
```
Comment (ilan-gold) [inline package.json]:
```suggestion
    "bench:h1": "./benchmark.js chrome_http1.csv --disable-http2 --ignore-https-errors --no-headless",
    "bench:h2": "./benchmark.js chrome_http2.csv --ignore-https-errors --no-headless",
```

**Review by ilan-gold — CHANGES_REQUESTED** [review]
I think the flag is wrong but otherwise works

**Pull request merged**